apps/server/src/shared/infra/identity-management/keycloak/service/keycloak-configuration.service.ts

```diff
@@ -100,6 +101,44 @@ export class KeycloakConfigurationService {
 		}
 	}
 
+	public async configureClient(): Promise<void> {
+		const kc = await this.kcAdmin.callKcAdminClient();
+		let id = (await kc.clients.find({ clientId: CLIENT_ID }))[0]?.id;
+		if (!id) {
+			({ id } = await kc.clients.create({
+				clientId: CLIENT_ID,
+				publicClient: false,
+			}));
+		}
+		const response = await kc.clients.generateNewClientSecret({ id });
```

Comment:
Can we give this const a more specific name than `response`? Maybe `generatedClientSecret` or something like that?

Reply:
Done.